### ISSUE TITLE
Improved UI for manually triggered jobs

### ISSUE THREAD

**Issue by jama22** [concourse/concourse#2233]
This is a UI improvement originally proposed by user `simonjohansson` on our forums, thread here: https://discuss.concourse-ci.org/t/improved-ui-for-manually-triggered-jobs/146

Opening an issue here as a way to keep the work to be done recorded

cc @Lindsayauchin

**Comment by jchesterpivotal** [comment]
I've previously pitched the idea of an "approvals resource" for this purpose. I feel like it would be a better fit into the existing model than a new kind of trigger.

**Comment by simonjohansson** [comment]
@jchesterpivotal a resource would indeed be a nice way of modeling this, but it would not solve the core issue I believe, in that it would not be clear in the UI when it needs a trigger?

**Comment by jchesterpivotal** [comment]
Well there you have me. But I can also argue this of any other triggering resource -- I'm wary of introducing special cases.

**Comment by simonjohansson** [comment]
@jchesterpivotal 
> I'm wary of introducing special cases.

Absolutely, I totally agree. My suggestion in the first post of the discourse would apply to any job that have a `trigger: false` and where an "upstream" job have passed and/or resource have a new version

**Comment by simonjohansson** [comment]
Howdie! 

Is there any update on when/if this is going to be worked on? 15 thumbs up and 5 hears seems to indicate that this is something many people want :)

**Comment by simonjohansson** [comment]
Ping pong. :)

**Comment by taylorsilva** [comment]
# Issue Bankruptcy

This issue is being closed as part of 2025's **Issue Bankruptcy** declaration. [See here for details](https://blog.concourse-ci.org/posts/2025-01-06-concourse-2024-review/)

If you think this issue still has merit then feel free to open a new issue and reference this one.
